proto/config/appconfig.proto

```diff
@@ -206,6 +206,10 @@ message AppInstanceConfig {
 
   // PatchEnvelopeRef is used to track all patch envelopes used by the app instance
   PatchEnvelopeRef patchRef = 23;
+
+  // allow to be discovered by other app instances in the same network
+  // instance. Default is false
```

Comment:
Comment is reversed; we want to grant the ability for an app instance to discover all the other app instances attached to its network instances. 
Unless @gkodali-zededa tells me otherwise.

Reply:
Yes. @eriknordmark is right.